pkg/jira/api.go

```diff
@@ -120,6 +120,47 @@ func CreateIssue(summary string) (string, error) {
 	return response.Key, nil
 }
 
+// AddIssueLabels updates Jira issue summary
+func AddIssueLabels(issueKey string, labels []string) ([]string, error) {
+	client := &http.Client{}
+
+	requestURL, err := getRequestURL(jiraOperationUpdateIssue, issueKey)
+	if err != nil {
+		return labels, err
+	}
+
+	var updateIssueLabels []UpdateIssueLabels
+	for _, label := range labels {
+		updateIssueLabels = append(updateIssueLabels, UpdateIssueLabels{Add: label}) // note the = instead of :=
+	}
+	formValues := UpdateIssueData{
```

Comment:
Do we need extra space after `UpdateIssueData` and before bracket?

Reply:
Good question :) As I can see for constructors there is no need for empty line before brackets :)